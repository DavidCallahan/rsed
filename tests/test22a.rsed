$text = " a b c a "
$new = replace "a" with "x" in $text
print $new
$new = replace all "a" with "x" in $text
print $new
