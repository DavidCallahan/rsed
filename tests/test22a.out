 x b c a 
 x b c x 
